src/ess/amor/normalization.py

```diff
@@ -54,7 +54,7 @@ def mask_events_where_supermirror_does_not_cover(
             theta(
                 sam.bins.coords["wavelength"],
                 sam.coords["pixel_divergence_angle"],
-                ref.coords["L2"],
+                sam.coords["L2"],
```

Comment:
Is this a bugfix? If yes, please add a test!

Reply:
Not a bugfix, they are equivalent, I just found the second version more clear.